pinot-spi/src/main/java/org/apache/pinot/spi/crypt/PinotCrypterFactory.java

```diff
@@ -60,12 +61,11 @@ public static void init(Configuration config) {
       LOGGER.info("Got crypter class name {}, full crypter path {}, starting to initialize", key, className);
 
       try {
-        PinotCrypter pinotCrypter = (PinotCrypter) Class.forName(className).newInstance();
+        PinotCrypter pinotCrypter = PluginManager.get().createInstance(className);
```

Comment:
For custom (non-open sourced) Pinot builds and deployments, the expectation is to update the build tooling to ensure that these plugins are provided correctly? For most other classes (except crypter, realtime ingestion), we likely only need them in standalone invocations, so build tooling doesn't need to change for them. Is my understanding correct?

Reply:
If the plugins are part of the build, then there is no need for any additional tooling.